bolt-cli/src/cli.rs

```diff
@@ -131,6 +138,29 @@ pub struct SendCommand {
     pub devnet_sidecar_url: Option<Url>,
 }
 
+/// Command for registering validators in the Bolt network.
+#[derive(Debug, Clone, Parser)]
+pub struct RegisterCommand {
+    /// The chain for which the delegation message is intended.
+    #[clap(long, env = "CHAIN", default_value = "holesky")]
+    pub chain: Chain,
+
+    #[clap(long, env = "MAX_COMMITTED_GAS_LIMIT", default_value_t = 10_000_000)]
+    pub max_committed_gas_limit: u32,
+
+    #[clap(long, env = "AUTHORIZED_OPERATOR")]
+    pub authorized_operator: Address,
+
+    #[clap(long, env = "PUBKEYS_PATH", default_value = "pubkeys.json")]
+    pub pubkeys_path: PathBuf,
+
+    #[clap(long, env = "ADMIN_PRIVATE_KEY")]
+    pub admin_private_key: String,
```

Comment:
I recommend actually parsing this as `B256` since it should automatically work from a hex string